lib/puma/app/status.rb

```diff
@@ -25,8 +28,13 @@ def call(env)
 
         # resp_type is processed by following case statement, return
         # is a number (status) or a string used as the body of a 200 response
+        command = env['PATH_INFO'][/\/([^\/]+)$/, 1]
+        if @enabled_commands && !@enabled_commands.include?(command)
+          return rack_response(404, "Command disabled", 'text/plain')
```

Comment:
If someone hits this with a non-existant command, it's not technically "disabled" but it is unavailable. 

```suggestion
          return rack_response(404, "Command #{command.inspect} unavailable", 'text/plain')
```

Also I like showing people what the code is seeing which might help them realize there's a typo or something else is off like random accidental whitespace.